### PR TITLE
Replace Helpdesk language with Data Support Team

### LINKED ISSUE #1545: Replace Helpdesk language with Data Support Team
Also move references to ODS/CDS to Appreciation: https://standard.open-contracting.org/latest/en/history/history_and_development/#appreciation :

* https://standard.open-contracting.org/latest/en/primer/next/?highlight=%22open%20data%20services%22#how-can-i-implement-the-ocds
* https://standard.open-contracting.org/latest/en/support/?highlight=%22open%20data%20services%22#ocds-helpdesk

For January 1, 2023.

### PR DESCRIPTION
**Pull request by jpmckinney**
closes #1545

### CODE REVIEW

**Comment by jpmckinney** [comment]
Had to edit base branch.

**Comment by jpmckinney** [comment]
There is one last occurrence on the homepage that I can fix by cherry-picking #1573 into 1.1-dev.

File: docs/support/index.md [hunk 1]
```diff
@@ -1,21 +1,19 @@
...
-If you are planning to publish or use OCDS data, then the OCDS Helpdesk is on-hand to offer advice and input, by:
+If you are planning to publish or use OCDS data, then the Data Support Team is on-hand to offer advice and input, by:
 
 * Helping you identify approaches for converting data from your existing systems to OCDS
 * Suggesting existing tools and services that can help you publish or use OCDS data
 * Providing guidance on mapping your data structures to OCDS
 * Giving you feedback on draft data files, and checking the structure of your data
 
-Helpdesk support is available in English and Spanish and delivered through collaboration with the [Open Data Services Cooperative](https://opendataservices.coop) in the United Kingdom and [Centro de Desarrollo Sostenible](http://www.cds.com.py) in Paraguay.
+Support is provided free of charge and is available in English, Russian and Spanish.
```
Comment (yolile) [inline docs/support/index.md]:
```suggestion
Support is provided free of charge and is available in English, Spanish and Russian.
```

File: docs/history/history_and_development.md [hunk 2]
```diff
@@ -28,6 +28,8 @@ Old tools that have been superseded or obsoleted by new tools can be found in th
 
 The OCDS was developed with input from many people, to whom we extend our thanks.
 
+From 2015 to 2022, [data support](../support/index) was delivered through collaboration with [Open Data Services Co-operative Limited](https://opendataservices.coop) in the United Kingdom and [Centro de Desarrollo Sostenible](http://www.cds.com.py) in Paraguay.
```
Comment (yolile) [inline docs/history/history_and_development.md]:
```suggestion
From 2015 to 2022, [data support](../support/index) was delivered through collaboration with [Open Data Services Co-operative Limited](https://opendataservices.coop) in the United Kingdom and [Centro de Desarrollo Sostenible](https://www.cds.com.py) in Paraguay.
```

**Review by yolile — APPROVED** [review]

**Comment by jpmckinney** [comment]
I've now cherry-picked the commits, so all mentions are done.

**Comment by jpmckinney** [comment]
Deployed

**Pull request merged**